tests/nightly/nightly-kickoff.sh

```diff
@@ -133,7 +133,7 @@ done
 # Setting the NSX test status to Not Implemented.
 nightlystatus[23]="TODO"
```

Comment:
Can you please update this to N/A. Currently we don't have support NSX for 6.0

nightlystatus[23]="N/A"